### PR TITLE
add css font-size examples

### PR DESCRIPTION
**Pull request by benji1304**
As per this issue 
https://github.com/mdn/interactive-examples/issues/563

### CODE REVIEW

**Comment by benji1304** [comment]
https://github.com/mdn/interactive-examples/pull/567/commits/c3b37aa083fb1498ab6d13fef3211bdb2c5655f0 for issue https://github.com/mdn/interactive-examples/issues/566 as discussed.

Hope this is getting better.

File: live-examples/css-examples/fonts/meta.json [hunk 1]
```diff
@@ -34,6 +34,14 @@
             "fileName": "font-weight.html",
             "title": "CSS Demo: font-weight",
             "type": "css"
+        },
+        "font-size": {
```
Comment (wbamberg) [inline live-examples/css-examples/fonts/meta.json]: This should be camel-cased - `fontSize` - and appear in alphabetical order in meta.json.

**Comment by benji1304** [comment]
Sure, sounds good. I'll take a look asap and update.

**Comment by wbamberg** [comment]
Sorry, I didn't see you had the updates for `color` as well. I'll review that now.

**Comment by wbamberg** [comment]
A couple of minor comments on `color`, neither of which you introduced, so it seems a little unfair to complain about :)

* coding style is to omit the leading `0` before a period, so `color: hsla(30, 100%, 50%, .3);` not `color: hsla(30, 100%, 50%, 0.3);`. We don't AFAIK have this documented anywhere :(

* the `rgb` value looks almost black, you might want to consider something a bit more obviously different.

**Comment by benji1304** [comment]
@wbamberg new PR on here with the fixes for color and font-size

File: live-examples/css-examples/fonts/font-size.html [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
-<div class="example-choice" initial-choice="true">
+<div class="example-choice">
 <pre><code class="language-css">font-size: 12px;</code></pre>
 <button type="button" class="copy hidden" aria-hidden="true">
     <span class="visually-hidden">Copy to Clipboard</span>
 </button>
 </div>  
 
-<div class="example-choice" initial-choice="true">
+<div class="example-choice"></div>
```
Comment (wbamberg) [inline live-examples/css-examples/fonts/font-size.html]: You have a rogue `</div>` here that's breaking the layout...

File: live-examples/css-examples/fonts/meta.json [hunk 3]
```diff
@@ -35,7 +35,7 @@
             "title": "CSS Demo: font-weight",
             "type": "css"
         },
-        "font-size": {
+        "fontSize": {
```
Comment (wbamberg) [inline live-examples/css-examples/fonts/meta.json]: This should appear in alphabetical order (after `fontFamily`).

**Comment by benji1304** [comment]
https://github.com/mdn/interactive-examples/pull/567/commits/550d5152d87b52ef4007ea9f10a85fae9c2b08d5 with updates

**Comment by wbamberg** [comment]
I've updated the MDN pages:
https://developer.mozilla.org/en-US/docs/Web/CSS/color
https://developer.mozilla.org/en-US/docs/Web/CSS/font-size
:)

**Pull request merged**